scripts/mitochondria_m2_wdl/MitochondriaPipeline.wdl

```diff
@@ -197,17 +205,19 @@ task SubsetBamToChrM {
     preemptible: select_first([preemptible_tries, 5])
   }
   output {
-    File output_bam = "${basename}.bam"
-    File output_bai = "${basename}.bai"
+    File output_bam = "~{basename}.bam"
+    File output_bai = "~{basename}.bai"
   }
 }
 
 task RevertSam {
-  File input_bam
-  String basename = basename(input_bam, ".bam")
+  input {
+    File input_bam
```

Comment:
If we call RevertSam through GATK we can use NIO here too, right?

Reply:
That's true, but by the time we subset down to just the mt bam everything is pretty quick. The NIO above helps because it's still on the full size hg38 wgs bam.